### PR TITLE
Fix Firefox references

### PR DESCRIPTION
**Pull request by fernandojsg**
Use release instead of nightly, and add reference to Vive support on macOS.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -109,10 +109,11 @@ <h3>
                         <span class="carrot"></span>
                         <div class="device-box-header">HTC Vive</div>
                         <span>
-                            Works with <a href="https://webvr.rocks/firefox">Firefox Nightly</a>,
-                            <a href="https://blog.mozvr.com/webvr-servo-architecture-and-latency-optimizations/">Servo</a>, and
+                            Works with <a href="https://webvr.rocks/firefox">Firefox</a>,
+                            <a href="https://blog.mozvr.com/webvr-servo-architecture-and-latency-optimizations/">Servo</a> and
```
Comment (cvan) [inline index.html]: might as well change this link to https://webvr.rocks/servo or https://blog.servo.org/2018/03/09/servo-and-mixed-reality/

File: index.html [hunk 2]
```diff
@@ -129,7 +130,7 @@ <h3>
                         <div class="device-box-header">Windows Mixed Reality headsets</div>
                         <span>
                             <a href="https://docs.microsoft.com/en-us/microsoft-edge/webvr/webvr-with-edge/">Microsoft Edge supports WebVR</a>. <a href="https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webvr/">Status: "The WebVR 1.1 API is fully supported."</a>.
-                            <a href="https://www.supermedium.com">Supermedium</a> also supports through SteamVR.
+                            <a href="https://webvr.rocks/firefox">Firefox</a> and <a href="https://www.supermedium.com">Supermedium</a> also support through SteamVR.
```
Comment (cvan) [inline index.html]: can you add this link for [SteamVR](https://docs.microsoft.com/en-us/windows/mixed-reality/enthusiast-guide/using-steamvr-with-windows-mixed-reality)?

Comment (cvan) [reply index.html]: `also support through` -> `are also supported with`?

File: index.html [hunk 3]
```diff
@@ -97,7 +97,7 @@ <h3>
                         <span class="carrot"></span>
                         <div class="device-box-header">Oculus Rift</div>
                         <span>
-                            Works with <a href="https://webvr.rocks/firefox">Firefox Nightly</a> and
+                            Works with <a href="https://webvr.rocks/firefox">Firefox</a> and
                             <a href="https://www.supermedium.com">Supermedium</a>
                             on Windows
```
Comment (cvan) [inline index.html]: nit: there should be a period `.` here

Comment (cvan) [reply index.html]: and probably elsewhere too - for consistency:

image

File: index.html [hunk 4]
```diff
@@ -109,10 +109,11 @@ <h3>
...
                         <div class="device-box-header">HTC Vive</div>
                         <span>
-                            Works with <a href="https://webvr.rocks/firefox">Firefox Nightly</a>,
-                            <a href="https://blog.mozvr.com/webvr-servo-architecture-and-latency-optimizations/">Servo</a>, and
+                            Works with <a href="https://webvr.rocks/firefox">Firefox</a>,
+                            <a href="https://blog.mozvr.com/webvr-servo-architecture-and-latency-optimizations/">Servo</a> and
                             <a href="https://www.supermedium.com">Supermedium</a>
-                            on Windows
+                            on Windows.
+                            <a href="https://nightly.mozilla.org/">Firefox Nightly</a> on macOS.
```
Comment (cvan) [inline index.html]: did you intend to add a `<br>` before this line?

Comment (cvan) [reply index.html]: also, you can use the Oculus Rift w/ Firefox via SteamVR (OpenVR) with macOS, right? I'm pretty certain it's not just the Vive. cc @blairmacintyre @kearwood?

File: index.html (hunk 1 above)
Comment (fernandojsg) [reply index.html]: Done thanks!

File: index.html (hunk 2 above)
Comment (fernandojsg) [reply index.html]: Sure, done.

File: index.html (hunk 3 above)
Comment (fernandojsg) [reply index.html]: Done

File: index.html (hunk 4 above)
Comment (fernandojsg) [reply index.html]: Added `<br>`. 
Re `Oculus` I wasn't sure about it. Waiting for their answer

File: index.html [hunk 5]
```diff
@@ -109,18 +109,19 @@ <h3>
                         <span class="carrot"></span>
                         <div class="device-box-header">HTC Vive</div>
                         <span>
-                            Works with <a href="https://webvr.rocks/firefox">Firefox Nightly</a>,
-                            <a href="https://blog.mozvr.com/webvr-servo-architecture-and-latency-optimizations/">Servo</a>, and
+                            Works with <a href="https://webvr.rocks/firefox">Firefox</a>,
+                            <a href="https://webvr.rocks/servo">Servo</a> and
                             <a href="https://www.supermedium.com">Supermedium</a>
-                            on Windows
+                            on Windows.<br/>
```
Comment (cvan) [inline index.html]: can you replace this `<br/>` with this:

```html
                         </span>
                         <span>
```

File: index.html [hunk 6]
```diff
@@ -129,7 +130,7 @@ <h3>
                         <div class="device-box-header">Windows Mixed Reality headsets</div>
                         <span>
                             <a href="https://docs.microsoft.com/en-us/microsoft-edge/webvr/webvr-with-edge/">Microsoft Edge supports WebVR</a>. <a href="https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webvr/">Status: "The WebVR 1.1 API is fully supported."</a>.
-                            <a href="https://www.supermedium.com">Supermedium</a> also supports through SteamVR.
+                            <a href="https://webvr.rocks/firefox">Firefox</a> and <a href="https://www.supermedium.com">Supermedium</a> are also supported with <a href="https://docs.microsoft.com/en-us/windows/mixed-reality/enthusiast-guide/using-steamvr-with-windows-mixed-reality">SteamVR</a>.
```
Comment (cvan) [inline index.html]: perfect 👍thanks!

**Pull request merged**